archivist_samples/testing/constants.py

```diff
@@ -0,0 +1,4 @@
+"""Constants
+"""
+
+USER_AGENT_PREFIX = "samples/"
```

Comment:
I find this arrangement rather confusing and potential brittle if it's repeated anywhere. Wat's the benefit of this indirection?